ocaml/forkexecd/test/fe_test.ml

```diff
@@ -109,14 +109,18 @@ let test_delay () =
   let start = Unix.gettimeofday () in
   let exe = Printf.sprintf "/proc/%d/exe" (Unix.getpid ()) in
   let args = ["sleep"] in
+  let timeout = 3.7 in
   try
-    Forkhelpers.execute_command_get_output ~timeout:4.0 exe args |> ignore ;
+    Forkhelpers.execute_command_get_output ~timeout exe args |> ignore ;
     failwith "Failed to timeout"
   with
   | Forkhelpers.Subprocess_timeout ->
-      Printf.printf "Caught timeout exception after %f seconds\n%!"
-        (Unix.gettimeofday () -. start) ;
-      ()
+      let elapsed = Unix.gettimeofday () -. start in
```

Comment:
I think the change is fine. For new code we like to use `Ptime.Span` for time spans.